bcml/gui.py

```diff
@@ -30,31 +30,40 @@
 class BcmlFrame(wx.Frame):
     mods = {}
     cemudir = ''
+    enable_controls = True
 
     def __init__(self, *args, **kwds):
         # begin wxGlade: BcmlFrame.__init__
-        kwds["style"] = kwds.get("style", 0) | wx.DEFAULT_FRAME_STYLE ^ wx.RESIZE_BORDER
+        kwds["style"] = kwds.get("style", 0) | wx.CAPTION | wx.MINIMIZE_BOX | wx.CLOSE_BOX ^ wx.RESIZE_BORDER
         wx.Frame.__init__(self, *args, **kwds)
         self.list_mods = wx.ListBox(self, wx.ID_ANY, choices=["<No mods installed>"])
-        self.button_install = wx.Button(self, wx.ID_ANY, "Install...")
-        self.button_uninstall = wx.Button(self, wx.ID_ANY, "Uninstall")
-        self.button_change = wx.Button(self, wx.ID_ANY, "Change Priority...")
-        self.button_update = wx.Button(self, wx.ID_ANY, "Update")
-        self.button_export = wx.Button(self, wx.ID_ANY, "Export...")
-        self.button_explore = wx.Button(self, wx.ID_ANY, "Explore...")
+        self.button_install_container = wx.Panel(self)
+        self.button_uninstall_container = wx.Panel(self)
+        self.button_change_container = wx.Panel(self)
+        self.button_update_container = wx.Panel(self)
+        self.button_export_container = wx.Panel(self)
+        self.button_explore_container = wx.Panel(self)
+        self.button_install = wx.Button(self.button_install_container, wx.ID_ANY, "Install...")
+        self.button_uninstall = wx.Button(self.button_uninstall_container, wx.ID_ANY, "Uninstall")
+        self.button_change = wx.Button(self.button_change_container, wx.ID_ANY, "Change Priority...")
+        self.button_update = wx.Button(self.button_update_container, wx.ID_ANY, "Update")
+        self.button_export = wx.Button(self.button_export_container, wx.ID_ANY, "Export...")
+        self.button_explore = wx.Button(self.button_explore_container, wx.ID_ANY, "Explore...")
```

Comment:
Here we add each of the buttons to their own panels, and later assign both the buttons and the panels the same tooltips so that in the case of the button being disabled the tooltip is still displayed.